scripts/utils/getBallotContract.ts

```diff
@@ -0,0 +1,22 @@
+import {ethers} from 'ethers';
+import {Ballot__factory} from './../../typechain-types/factories/Ballot__factory';
+import * as dotenv from "dotenv";
+import {Ballot} from '../../typechain-types';
+
+dotenv.config();
+
+function getBallotContract(contractAddress: string): Ballot {
+  if (!process.env.GOERLI_PRIVATE_KEY) {
+    throw new Error("GOERLI_PRIVATE_KEY not found");
+  }
+
+  const provider = new ethers.providers.AlchemyProvider("goerli", process.env.ALCHEMY_API_KEY);
+  const signer = new ethers.Wallet(process.env.GOERLI_PRIVATE_KEY, provider);
+
+  const ballotContractFactory = new Ballot__factory(signer);
+  const ballot = ballotContractFactory.attach(contractAddress)
+
+  return ballot
+}
+
+export default getBallotContract
```

Comment:
Could we make this a more generic file, eg. `getContract()`?

If we pass `contractName `as we do for the `deploy `command, maybe we could match it against the typechain and factories imports to make the function type and contractFactory types dynamic.

Reply:
I think we should just merge this now and we can get back to my suggestion later if relevant.

Reply:
I'll look into this refactor if I have time.